### ISSUE TITLE
Define Resource Descriptors Function: Support multiple nodes to populate description and map pop panels

### ISSUE THREAD

**Issue by dwuthrich** [archesproject/arches#2720]
### Bug Report

As a data designer, i need to be able to populate Arches map pop-up content and the search description content for a resource from more than a single node.

#### Expected Behavior

Allow a designer to select from multiple nodes to define the content to be displayed in the map popup and the display description panels

#### Steps to Reproduce (for bugs)

1.
2.
3.
4.

#### Your Arches Information

* Version used:
* Browser Name and version:
* Operating System and version (desktop or mobile):
* Link to your Arches Install (optional):

**Comment by mradamcox** [comment]
I agree with this issue, because currently the limitations of this function can dictate database design. However, just to clarify, I think you mean **nodegroup** instead of node... Currently the function allows you to select a nodegroup, and then use as many nodes from within that group as you want. However, it does not allow you to aggregate nodes from different node groups.

**Comment by dwuthrich** [comment]
This is still a desirable new capability

**Comment by chiatt** [comment]
The assumption that primary descriptors are sourced from one nodegroup is baked into other parts of Arches, so this ticket should be addressed after #8281, or at least in combination with it.

**Comment by chiatt** [comment]
Closing as a duplicate of #4312, which although more recent, has more discussion
